conjure-java-jaxrs-client/build.gradle

```diff
@@ -22,13 +22,17 @@ dependencies {
 
     implementation 'com.palantir.tritium:tritium-registry'
 
+    // include both the old and the new
+    implementation 'javax.ws.rs:javax.ws.rs-api'
+    implementation 'jakarta.ws.rs:jakarta.ws.rs-api'
```

Comment:
Should we include both APIs here? I’d assumed we wanted to include neither such that we would only see javax annotations in classpaths that need it? (Once we’ve managed to decouple it from conjure-lib, anyhow)

Reply:
Not needed anymore.